Makefile

```diff
@@ -99,7 +99,7 @@ setup-kong-build-tools:
 	-rm -rf $(KONG_BUILD_TOOLS_LOCATION)
 	-git clone https://github.com/Kong/kong-build-tools.git $(KONG_BUILD_TOOLS_LOCATION)
 	cd $(KONG_BUILD_TOOLS_LOCATION); \
-	git reset --hard $(KONG_BUILD_TOOLS); \
+	git reset --hard && git checkout $(KONG_BUILD_TOOLS); \
```

Comment:
Why is this needed?

Reply:
git can't reset to branches, only to tags. With this change it's possible to checkout an unreleased kong-build-tools version.

Reply:
I thought it would work if you prefixed with `origin/` but maybe I'm wrong